src/test/java/com/arangodb/example/ssl/SslExample.java

```diff
@@ -55,6 +56,35 @@
 	@Test
 	@Ignore
 	public void connect() throws Exception {
+		final ArangoDB arangoDB = new ArangoDB.Builder()
+				.host("localhost", 8529)
+				.password("test")
+				.useSsl(true)
+				.sslContext(createSslContext())
+				.useProtocol(Protocol.HTTP_JSON)
+				.build();
+		final ArangoDBVersion version = arangoDB.getVersion();
+		assertThat(version, is(notNullValue()));
+		System.out.println(version.getVersion());
+	}
+
+	@Test

```

Comment:
Do. we have actual tests or just examples?

Reply:
despite the name, this is a test that only passes if
```
assertThat(version, is(notNullValue()));
```
This is enough, since the underlying layers would throw an error and make the test fail in case of ssl connection problems.

Nevertheless due to the current test architecture we do not run this test automatically (it is annotated with `@Ignore`), since it requires a db server configured with ssl. 

On the other side, such kind of tests can be executed in the `next` branch, where the test databases are started programmatically from Java code as docker containers, therefore ssl certs can be easily shared.